public/scripts/main.js

```diff
@@ -422,3 +423,24 @@ window.addEventListener('focus', function (e) {
     onSuccess();
   }
 });
+
+function createUserAnswer(questionNo, isCorrect, elapsedTimeMs){
+  let requestBody = {
+    userId: 'guest',
+    isCorrect: isCorrect,
+    elapsedTimeMs: elapsedTimeMs
+  }
+  
+  fetch(document.URL + 'user/answers/question/' + questionNo, {
```

Comment:
This works fine. For future reference, you can also omit the entire hostname, etc and just use a relative URL, e.g. `fetch('/user/...')`